spec/features/applications/create_spec.rb

```diff
@@ -35,4 +35,18 @@
     #And I see an indicator that this application is "In Progress"
     expect(page).to have_content("In Progress")
   end
+
```

Comment:
Great test - we may need to refactor line 48 since an HTTP request isn't supposed to be created when the form fails validation